qbitunregistered.py

```
@@ -68,60 +87,50 @@
     try:
-        file_handler = logging.FileHandler(log_file, mode='a')
-        file_handler.setFormatter(logging.Formatter(
-            '%(asctime)s %(levelname)s: %(message)s',
-            datefmt='%Y-%m-%d %H:%M:%S'
-        ))
+        file_handler = logging.FileHandler(log_file, mode="a")
+        file_handler.setFormatter(logging.Formatter("%(asctime)s %(levelname)s: %(message)s", datefmt="%Y-%m-%d %H:%M:%S"))
         log_handlers.append(file_handler)
     except (KeyboardInterrupt, SystemExit):
         raise
     except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)